src/content/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api.mdx

```diff
@@ -624,6 +634,35 @@ The following APIs provide additional functionality, such as setting app server
         When inside a transaction, the last call to `noticeError` wins. Only 1 error will be reported per transaction.
       </td>
     </tr>
+##here
+
+    <tr>
+      <td>
+         Classes that implement the [`ErrorGroupCallback`](https://newrelic.github.io/java-agent-api/javadoc/index.html?com/newrelic/api/agent/NewRelic.html) interface are used to generate grouping keys for errors that will be sent to
+         New Relic.  These keys will be used to group error messages in the Errors Inbox UI.
```

Comment:
Seems like we want this to read similarly to what's there already too. eg: since it's the "If you want to..." column, then maybe have it read something like:

"Group error messages with your own custom defined key"

With the "custom defined key" linking out to that other page

Reply:
Also changed naming to "fingerprint" instead of key because that's what we have in the spec.

Reply:
Done